### ISSUE TITLE
Mo-doc produces unexpected Debug output when running

### ISSUE THREAD

**Issue by ByronBecker** [dfinity/motoko#3306]
Using the `mo-doc` tool with motoko version 0.6.28 (downloaded from binary release), I'm seeing logging to stdout that I'm not sure is supposed to be there, or if it is doesn't really tell me what I should do next?

The documentation successfully compiles (i.e. I don't see any errors in the output), and there's a fair amount of documentation that gets generated (9 html files) with just these two `(ObjP...)` outputs being logged, making me think it's a stray logging output somewhere.

```
% $(shell vessel bin)/mo-doc
[INFO] vessel.dhall
.vessel/.bin/0.6.28/mo-doc
(ObjP
  (canisterId (AnnotP (VarP canisterId) (PathT (IdH Principal))))
  (settings
    (AnnotP (VarP settings) (PathT (DotH (IdH InterfaceSpec) CanisterSettings)))
  )
)
(ObjP
  (canisterId (AnnotP (VarP canisterId) (PathT (IdH Principal))))
  (wasmModule (AnnotP (VarP wasmModule) (PathT (IdH Blob))))
  (args (AnnotP (VarP args) (PathT (IdH Blob))))
)
```

**Comment by ggreif** [comment]
```
$ git grep -in arrange.pat
src/docs/extract.ml:111:        Wasm.Sexpr.print 80 (Arrange.pat pat);
```

**Comment by ggreif** [comment]
On it!

**Comment by ggreif** [comment]
This will  be fixed in `moc` 0.6.30. Thanks for the report @ByronBecker!
